org-formation/organization.yaml

```diff
@@ -566,6 +567,21 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  GenieProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: genie-prod
+      RootEmail: aws-genie-prod@sagebase.org
+      Alias: org-sagebase-genie-prod
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: DPE
+        Project: Genie
+        AccountOwner: thomas.yu@sagebase.org
+        CostCenter: Genie-AACR BPC / 312100
```

Comment:
these tags are different than `RecoverDevAccount`, is that intentional?

Reply:
Yes, this is the correct cost center for the Genie BPC Shiny app', as per @thomasyu888 .